test/addons/rook-cephfs/storage-class.yaml

```diff
@@ -11,7 +11,7 @@ kind: StorageClass
 metadata:
   name: rook-cephfs-$fsname
   labels:
-    ramendr.openshift.io/storageid: rook-cephfs-$cluster-1
+    ramendr.openshift.io/storageid: rook-cephfs-$fsname-$cluster-1
```

Comment:
I wonder why we don't use the same method - the storage class name. Using the same way to customize will make the code simpler and more uniform.

Reply:
updated.